README.md

```diff
@@ -243,13 +257,32 @@ For a step-by-step guide on deploying production server with static files,
    ]
    ```
 
-3. Ensure that `BASE_DIR` setting is defined in settings.py:
+3. `BASE_DIR` setting is required. Ensure that it is defined in settings.py:
 
    ```py
    BASE_DIR = Path(__file__).resolve().parent.parent
    ```
 
-4. Modify `TEMPLATES` section of settings.py as follows:
+4. Add / modify `COMPONENTS.dirs` so django_components knows where to  find component HTML, JS and CSS files:
+
+   ```python
+   COMPONENTS = {
+       "dirs": [
+            ...,
+            os.path.join(BASE_DIR, "components"),
+        ],
+   }
+   ```
+
+   If `COMPONENTS.dirs` is omitted or empty, django-components will by default look for a top-level `/components` directory,
+   `{BASE_DIR}/components`.
+
+   Irrespective of `COMPONENTS.dirs`, django_components will also load components from app-level `/components` directories, e.g. `my-app/components/`.
+
+   NOTE: The input to `COMPONENTS.dirs` are the same as for `STATICFILES_DIRS`, and the paths must be full paths. [See Django docs](https://docs.djangoproject.com/en/5.0/ref/settings/#staticfiles-dirs).
+
+
+5. Next, to make Django load component HTML files as Django templates, modify `TEMPLATES` section of settings.py as follows:
```

Comment:
Since the defaults are so good, I think we can remove this from the basic setup. Everyone should be good with the defaults.

Reply:
We can't, we still need to insert `django_components.template_loader.Loader` so we can let Django handle the loading and caching of the HTML templates.